server/services/transport/small-webrtc.mdx

```diff
@@ -0,0 +1,454 @@
+---
+title: "SmallWebRTCTransport"
+description: "A lightweight WebRTC transport for peer-to-peer audio and video communication in Pipecat"
+---
+
+## Overview
+
+`SmallWebRTCTransport` enables peer-to-peer WebRTC connections between clients and your Pipecat application. It implements bidirectional audio and video streaming using WebRTC for real-time communication.
+
+This transport is intended for lightweight implementations, particularly for local development and testing.
+
+<Warning>
+  `SmallWebRTCTransport` is best used for testing and development. For
+  production deployments with scale, consider using
+  [DailyTransport](/server/services/transport/daily) or LiveKitTransport, as
```

Comment:
Maybe we should remove the mention of LiveKitTransport? I don’t think it’s mentioned anywhere else in our docs.